Support/support_center_controller.rst

```diff
@@ -270,22 +270,22 @@ How can I restore my password when I lost or forget it ?
 
 In case if you lost or forget the password to AVX console, please use next steps to repair it:
 
-1.Press “Forgot password” from the login page
+1. Press “Forgot password” from the login page
 |login_page|
 
-2.Check email and find the one time token inside:
+2. Check email and find the one time token inside:
 Message format is :
 <<ONE TIME TOKEN>> is the one time Aviatrix token from controller <<IP ADDR AVX CONTROLLER>> and is valid for 15 minutes.
 Please pay attention that the token expires in 15 minutes. If you repeatedly get this and think that this is being done by someone with malicious intent, you can restrict the IP's allowed to access your controller through AWS's Security Groups
 
-3.Enter Access Token in Account Verification window:
+3. Enter Access Token in Account Verification window:
 |verification_window|
 
-4.Type new password for the admin user:
+4. Type new password for the admin user:
 
```

Comment:
Item 4 has a small format issue, from html format, you will see there's a space before character 4.
I think you can improve this by moving |admin_user| up and close to 4. Type new password for the admin user: